src/examples/zkapps/token_with_proofs.ts

```diff
@@ -55,9 +58,13 @@ class TokenContract extends SmartContract {
   @method sendTokens(
     senderAddress: PublicKey,
     receiverAddress: PublicKey,
-    callback: Experimental.Callback
+    callback: Experimental.Callback<any>
   ) {
-    let senderParty = Experimental.partyFromCallback(this, callback, true);
+    let senderParty = Experimental.partyFromCallback(
+      this,
+      [undefined],
```

Comment:
not a dummy party, but one that can have any children. so we're not constraining what children the party has (we're just witnessing the children's hash as a Field, instead of computing it in the circuit)